bumpversion.sh

```diff
@@ -10,10 +10,13 @@ bumpversion_build() {
 
 bumpversion_release() {
   bump2version patch
-  VERSION=$(bump2version --list --tag --commit --allow-dirty release | grep -oP '^new_version=\K.*$')
-  git push origin $RELEASE_BRANCH
+  git config user.name "aodn-ci-build"
+  git config user.email "info@aodn.edu.au"
+  VERSION=$(bump2version --list --commit --allow-dirty release | grep -oP '^new_version=\K.*$')
+  git fetch origin
```

Comment:
This looks like its going down the wrong path... the cloning has already been performed, and therefore the git "environment" already setup, so this script shouldn't be doing anything like resets or configuring git.

Reply:
The git multibranch pipeline plugin only does a shallow clone, just getting the latest code into the workspace, which isn't enough to do something like tagging.

The only other place to configure the git committer details would be the dockerfile. Imo it's better to set them here since it _is_  bumpversion that's doing the committing and the dockerfile should be more generic than that: it's not _necessarily_ being run by a our ci build server.

Reply:
I definitely wasn't suggesting the Dockerfile! More like in the config of Jenkins itself.

I don't understand why this wasn't a problem earlier... pretty sure this was tested against the (now deleted?) jenkins-tag-promotion repo and worked at the time. We should be doing this testing against a test repo in any case if there are credentials/config problems.